assignments/arrays.js

```diff
@@ -65,30 +65,68 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
 console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
```

Comment:
This console.log shows up in the browser. Since it's an example and you're not using it, either comment it out or delete it altogether -- don't want it loitering.